tests/test_mod.py

```diff
@@ -11,6 +11,23 @@
 from sentinelsat.sentinel import (InvalidChecksumError, SentinelAPI, SentinelAPIError, convert_timestamp, format_date,
                                   get_coordinates, md5_compare)
 
+_small_query = dict(
+        area='0 0,1 1,0 1,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 1, 2))
+
+_large_query = dict(
+        area='0 0,0 10,10 10,10 0,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 12, 31))
+
+_api_auth = dict(
```

Comment:
Good idea, @valgur. Maybe we could use [`skipif`](http://doc.pytest.org/en/latest/skipping.html#id1) for that, e.g. testing whether `_api_auth is None`. However, this would mean that these tests are also (silently?) skipped if the environment variables are not declared by mistake.